src/payment/strategies/mollie/mollie-payment-strategy.ts

```diff
@@ -27,19 +28,27 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
     private _verificationCodeElement?: MollieElement;
     private _expiryDateElement?: MollieElement;
 
+    private _hostedForm?: HostedForm;
+    private _shouldRenderHostedForm?: boolean;
```

Comment:
Since we are setting these 2 in tandem, should we just use the `_hostedForm`? If present, that means render it.

Reply:
yeap sure